two_factor/gateways/twilio/gateway.py

```diff
@@ -51,8 +53,15 @@ def make_call(self, device, token):
                                  from_=getattr(settings, 'TWILIO_CALLER_ID'),
                                  url=uri, method='GET', timeout=15)
 
-    def send_sms(self, device, token):
-        body = gettext('Your authentication token is %s') % token
+    def send_sms(self, device, token, sms_message):
```

Comment:
This method shouldn't require any extra parameters.

Reply:
done